appdev.Dockerfile

```diff
@@ -1,25 +1,22 @@
-FROM buildpack-deps:focal
+FROM ubuntu:focal
 
 ### base ###
+ENV DEBIAN_FRONTEND=noninteractive
 RUN yes | unminimize \
     && apt-get install -yq \
```

Comment:
Per the [tips article](https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34), I also tried to use `--no-install-recommends` on `apt-get install`, but this also started failing with some difficult error messages. In any case, I've shaved > 1 GB off of the un-compressed codespace and everything seems to be working find (e.g. I can `rails g scaffold post name` and CRUD)